ha/setup/ha_setup.py

```diff
@@ -633,7 +634,37 @@ def process(self):
         """
         Process reset command.
         """
-        pass # TBD
+        Log.info("Processing reset command")
+        try:
+            # create new version of log file
+            services = ["pcsd", "corosync", "cortx_ha_log.conf", "pacemaker"]
+            for service in services:
+                self._execute.run_cmd(f"logrotate --force /etc/logrotate.d/{service}")
+
+            older_logs = []
+            log_dirs = [const.COROSYNC_LOG, const.RA_LOG_DIR, "/var/log/pcsd"]
+            for log_dir in log_dirs:
+                log_list = self._execute.run_cmd(f"ls -I *.log -I *.xml {log_dir}")
+                logs = log_list[0].splitlines()
+                for log_file in logs:
+                    older_logs.append(os.path.join(log_dir, log_file))
+
+            pacemaker_log_list = self._execute.run_cmd("find /var/log -name pacemaker* -not -name *.log")
```

Comment:
Again, we should use commands if we don't find a python package for this. Use os module.

Reply:
Used python os module here